docs/package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@wmde/wikit-docs",
-  "version": "3.0.0-alpha.4",
+  "version": "3.0.0-alpha.5",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "@wmde/wikit-docs",
-      "version": "3.0.0-alpha.4",
+      "version": "3.0.0-alpha.5",
```

Comment:
This (and the equivalent line in the other two lockfiles) didn’t happen before – probably due to the new Lerna version. Seems like a good thing, though.